### PR TITLE
fix: include cancel handler in ServiceLauncherModal

### PR DESCRIPTION
**Pull request by agatha197**
### TL;DR

This pull request introduces a change to the `onCancel` handling in the `ServiceLauncherModal` to resolve the issue of the modify service modal popping up when clicking the start service button.

### What changed?
In the `ServiceListPage`, the `onRequestClose` method is called to set the `editingModelService` to null and remove the `onCancel` prop. 

In the `ServiceLauncherModal`, the `onCancel` behavior has been changed to call `onRequestClose` when triggered.

### How to test?

1.  Navigate to the serving page.
2. Click the gear icon in the Control column and close the modal with the close button at the bottom (not the x button).
3. Click Create new service button.
4. The create modal should appear.

---

resolves https://github.com/lablup/backend.ai-webui/issues/2378

**Checklist:** (if applicable)

- [x] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [x] Specific setting for review (eg., KB link, endpoint or how to setup) : 10.100.64.15
- [x] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by agatha197** [comment]
* **#2379** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2379?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @agatha197 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by yomybaby — CHANGES_REQUESTED** [review]
```
interface ServiceLauncherProps extends Omit<BAIModalProps, 'onOk' | 'onCancel'> {
```
Please update type defnition to fix typo and omit `onCancel`.

File: react/src/components/ServiceLauncherModal.tsx [hunk 1]
```diff
@@ -84,7 +84,8 @@ interface ServiceCreateType {
   config: ServiceCreateConfigType;
 }
 
-interface ServiceLauncherProps extends Omit<BAIModalProps, 'onOK'> {
+interface ServiceLauncherProps
+  extends Omit<BAIModalProps, 'onOK' | 'onCancel'> {
```
Comment (yomybaby) [inline react/src/components/ServiceLauncherModal.tsx]:
```suggestion
  extends Omit<BAIModalProps, 'onOk' | 'onCancel'> {
```
typo

**Review by yomybaby — CHANGES_REQUESTED** [review]
